querydsl-docs/src/main/docbook/ko-KR/content/tutorials/scala.xml

```diff
@@ -171,7 +171,7 @@ exporter.export(connection.getMetaData)
 query().from(employee).select(employee.firstName, employee.lastName)
 ]]></programlisting>
 
-      <para>다음과 같이 Employee 또는 QEmployee의 companio 객체를 사용할 수 있다.</para>
+      <para>다음과 같이 Employee 또는 QEmployee의 companion 객체를 사용할 수 있다.</para>
```

Comment:
Haha. Since it references "employee" I though that maybe they meant "company" so I popped it into google translate and companion made the most sense. I could be totally wrong.